vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -1184,15 +1199,31 @@ void WindowsTextInputComponentView::OnTextUpdated() noexcept {
   data.mostRecentEventCount = m_nativeEventCount;
 
   m_state->updateState(std::move(data));
+  std::string richEditText = GetTextFromRichEdit();
```

Comment:
For large contents of TextInput, this call could involve copying quite a decent sized string.  Looks like we only need this if m_eventEmitter && !m_comingFromJS.  -- Any chance we can refactor a little to avoid that?

Reply:
good catch, I didn't think about that! refactored the code to only call getText in the if statement